src/Components/Login.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { Redirect } from 'react-router-dom';
+import userLoginFetch from '../actions/actions.js';
+import userLogin from '../actions/login.js';
+import './Login.css';
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { 
+      authenticated: null,
```

Comment:
is authenticated useful?

Reply:
I had it at first but if we're using tokens, I guess not

Reply:
Yeah I think I had it too we can leave it in for now.